src/main/java/be/seeseemelk/mockbukkit/entity/EntityEquipmentMock.java

```diff
@@ -71,30 +72,38 @@ public void setItem(@NotNull EquipmentSlot slot, @Nullable ItemStack item, boole
 	}
 
 	@Override
-	public ItemStack getItem(@NotNull EquipmentSlot slot)
+	public @NotNull ItemStack getItem(@NotNull EquipmentSlot slot)
 	{
+		ItemStack itemStack;
 		switch (slot)
 		{
 		case CHEST:
-			return getChestplate();
+			itemStack = getChestplate();
+			break;
 		case FEET:
-			return getBoots();
+			itemStack = getBoots();
+			break;
 		case HAND:
-			return getItemInMainHand();
+			itemStack = getItemInMainHand();
+			break;
 		case HEAD:
-			return getHelmet();
+			itemStack = getHelmet();
+			break;
 		case LEGS:
-			return getLeggings();
+			itemStack = getLeggings();
+			break;
 		case OFF_HAND:
-			return getItemInOffHand();
+			itemStack = getItemInOffHand();
+			break;
 		default:
 			// This should never be reached unless Mojang adds new slots
 			throw new UnimplementedOperationException("EquipmentSlot '" + slot + "' has no implementation!");
 		}
+		return itemStack == null ? new ItemStack(Material.AIR) : itemStack;
```

Comment:
This implementation is redundant.
There is no point in storing the variable when we can just return it immediately.